src/duk_debugger.h

```diff
@@ -41,6 +42,7 @@
 #define DUK_DBG_CMD_DETACH        0x1f
 #define DUK_DBG_CMD_DUMPHEAP      0x20
 #define DUK_DBG_CMD_GETBYTECODE   0x21
+#define DUK_DBG_CMD_APPREQUEST    0x23
```

Comment:
Ok, so maybe change this to 0x22 and I'll make the object inspection 0x23. This will probably be in master first.